firmware/standalone/pacman/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ include(CheckCXXCompilerFlag)
 project(pacman_app)
 
 # Compiler options here.
-set(USE_OPT "-Os -g --specs=nano.specs")
+set(USE_OPT "-Os -g --specs=nano.specs --specs=nosys.specs")
```

Comment:
can we really use two specs, or simply the second overrides the first? nosys seems to be correct though, but @bernd-herzog should know better than I am 😇

Reply:
Ok digged a little, there can be multiple specs, but they might override each others individual directives and are processed in order

Reply:
The "application" cmake file already had these two specs, so we've been doing it that way for awhile btw.